From e028b58957b80424d719f14b097299926eb0e0c4 Mon Sep 17 00:00:00 2001
From: "K.Takata" <kentkt@csc.jp>
Date: Sat, 13 Oct 2018 21:50:16 +0900
Subject: [PATCH 31/31] nsis: Comment out unused translation messages

They were needed for gpwen's version. Currently, they are not used.
---
 nsis/lang/danish.nsi      |  44 +++++++--------
 nsis/lang/dutch.nsi       | 108 ++++++++++++++++++------------------
 nsis/lang/english.nsi     | 110 ++++++++++++++++++-------------------
 nsis/lang/german.nsi      | 108 ++++++++++++++++++------------------
 nsis/lang/italian.nsi     | 112 +++++++++++++++++++-------------------
 nsis/lang/japanese.nsi    | 106 ++++++++++++++++++------------------
 nsis/lang/simpchinese.nsi | 102 +++++++++++++++++-----------------
 nsis/lang/tradchinese.nsi | 102 +++++++++++++++++-----------------
 8 files changed, 396 insertions(+), 396 deletions(-)

diff --git a/nsis/lang/danish.nsi b/nsis/lang/danish.nsi
index aad6586e4..1dc18848c 100644
--- a/nsis/lang/danish.nsi
+++ b/nsis/lang/danish.nsi
@@ -30,7 +30,7 @@ LangString ^UninstallCaption ${LANG_DANISH}         "$(^Name) Uninstall"
 # MUI Configuration Strings                                               {{{1
 ##############################################################################
 
-LangString str_dest_folder          ${LANG_DANISH}     "Destinationsmappe (skal slutte med $\"vim$\")"
+#LangString str_dest_folder          ${LANG_DANISH}     "Destinationsmappe (skal slutte med $\"vim$\")"
 
 LangString str_show_readme          ${LANG_DANISH}     "Vis README efter installationen er gennemført"
 
@@ -67,17 +67,17 @@ LangString str_desc_desktop         ${LANG_DANISH}     "Opret ikoner til gVim-ek
 LangString str_section_start_menu   ${LANG_DANISH}     "I Programmer-mappen i menuen Start"
 LangString str_desc_start_menu      ${LANG_DANISH}     "Tilføj Vim i Programmer-mappen i menuen Start."
 
-LangString str_section_quick_launch ${LANG_DANISH}     "I værktøjslinjen Hurtig start"
-LangString str_desc_quick_launch    ${LANG_DANISH}     "Tilføj Vim-genvej i værktøjslinjen Hurtig start."
+#LangString str_section_quick_launch ${LANG_DANISH}     "I værktøjslinjen Hurtig start"
+#LangString str_desc_quick_launch    ${LANG_DANISH}     "Tilføj Vim-genvej i værktøjslinjen Hurtig start."
 
 LangString str_section_edit_with    ${LANG_DANISH}     "Tilføj Vim-genvejsmenu"
 LangString str_desc_edit_with       ${LANG_DANISH}     "Tilføj Vim til listen i $\"Åbn med...$\"-genvejsmenuen."
 
-LangString str_section_edit_with32  ${LANG_DANISH}     "32-bit-version"
-LangString str_desc_edit_with32     ${LANG_DANISH}     "Tilføj Vim til listen i $\"Åbn med...$\"-genvejsmenuen for 32-bit-programmer."
+#LangString str_section_edit_with32  ${LANG_DANISH}     "32-bit-version"
+#LangString str_desc_edit_with32     ${LANG_DANISH}     "Tilføj Vim til listen i $\"Åbn med...$\"-genvejsmenuen for 32-bit-programmer."
 
-LangString str_section_edit_with64  ${LANG_DANISH}     "64-bit-version"
-LangString str_desc_edit_with64     ${LANG_DANISH}     "Tilføj Vim til listen i $\"Åbn med...$\"-genvejsmenuen for 64-bit-programmer."
+#LangString str_section_edit_with64  ${LANG_DANISH}     "64-bit-version"
+#LangString str_desc_edit_with64     ${LANG_DANISH}     "Tilføj Vim til listen i $\"Åbn med...$\"-genvejsmenuen for 64-bit-programmer."
 
 LangString str_section_vim_rc       ${LANG_DANISH}     "Opret standardkonfiguration"
 LangString str_desc_vim_rc          ${LANG_DANISH}     "Opret en standardkonfigurationsfil (_vimrc) hvis der ikke allerede findes en."
@@ -114,39 +114,39 @@ LangString str_desc_rm_rootdir      ${LANG_DANISH}     "Fjern Vim-rodmappen. Den
 # Messages                                                                {{{1
 ##############################################################################
 
-LangString str_msg_too_many_ver  ${LANG_DANISH}     "Fandt $vim_old_ver_count Vim-versioner på dit system.$\r$\nInstallationsguiden kan højst håndtere ${VIM_MAX_OLD_VER}-versioner.$\r$\nFjern venligst nogle versioner og start igen."
+#LangString str_msg_too_many_ver  ${LANG_DANISH}     "Fandt $vim_old_ver_count Vim-versioner på dit system.$\r$\nInstallationsguiden kan højst håndtere ${VIM_MAX_OLD_VER}-versioner.$\r$\nFjern venligst nogle versioner og start igen."
 
-LangString str_msg_invalid_root  ${LANG_DANISH}     "Ugyldig installationssti: $vim_install_root!$\r$\nDen skal slutte med $\"vim$\"."
+#LangString str_msg_invalid_root  ${LANG_DANISH}     "Ugyldig installationssti: $vim_install_root!$\r$\nDen skal slutte med $\"vim$\"."
 
-LangString str_msg_bin_mismatch  ${LANG_DANISH}     "Uoverensstemmelse i binære sti!$\r$\n$\r$\nForventede at den binære sti var $\"$vim_bin_path$\",$\r$\nmen systemet indikerer at den binære sti er $\"$INSTDIR$\"."
+#LangString str_msg_bin_mismatch  ${LANG_DANISH}     "Uoverensstemmelse i binære sti!$\r$\n$\r$\nForventede at den binære sti var $\"$vim_bin_path$\",$\r$\nmen systemet indikerer at den binære sti er $\"$INSTDIR$\"."
 
-LangString str_msg_vim_running   ${LANG_DANISH}     "Vim kører stadig på dit system.$\r$\nLuk venligst alle instanser af Vim inden du fortsætter."
+#LangString str_msg_vim_running   ${LANG_DANISH}     "Vim kører stadig på dit system.$\r$\nLuk venligst alle instanser af Vim inden du fortsætter."
 
-LangString str_msg_register_ole  ${LANG_DANISH}     "Forsøger at registrere Vim med OLE. Der er ingen meddelelse til at indikere om det virker eller ej."
+#LangString str_msg_register_ole  ${LANG_DANISH}     "Forsøger at registrere Vim med OLE. Der er ingen meddelelse til at indikere om det virker eller ej."
 
-LangString str_msg_unreg_ole     ${LANG_DANISH}     "Forsøger at afregistrere Vim med OLE. Der er ingen meddelelse til at indikere om det virker eller ej."
+#LangString str_msg_unreg_ole     ${LANG_DANISH}     "Forsøger at afregistrere Vim med OLE. Der er ingen meddelelse til at indikere om det virker eller ej."
 
-LangString str_msg_rm_start      ${LANG_DANISH}     "Afinstallerer følgende version:"
+#LangString str_msg_rm_start      ${LANG_DANISH}     "Afinstallerer følgende version:"
 
-LangString str_msg_rm_fail       ${LANG_DANISH}     "Kunne ikke afinstallere følgende version:"
+#LangString str_msg_rm_fail       ${LANG_DANISH}     "Kunne ikke afinstallere følgende version:"
 
-LangString str_msg_no_rm_key     ${LANG_DANISH}     "Kan ikke finde registreringsdatabasenøgle for afinstallationsguiden."
+#LangString str_msg_no_rm_key     ${LANG_DANISH}     "Kan ikke finde registreringsdatabasenøgle for afinstallationsguiden."
 
-LangString str_msg_no_rm_reg     ${LANG_DANISH}     "Kan ikke finde afinstallationsguiden fra registreringsdatabasen."
+#LangString str_msg_no_rm_reg     ${LANG_DANISH}     "Kan ikke finde afinstallationsguiden fra registreringsdatabasen."
 
-LangString str_msg_no_rm_exe     ${LANG_DANISH}     "Kan ikke tilgå afinstallationsguide."
+#LangString str_msg_no_rm_exe     ${LANG_DANISH}     "Kan ikke tilgå afinstallationsguide."
 
-LangString str_msg_rm_copy_fail  ${LANG_DANISH}     "Kunne ikke kopiere afinstallationsguide til midlertidig mappe."
+#LangString str_msg_rm_copy_fail  ${LANG_DANISH}     "Kunne ikke kopiere afinstallationsguide til midlertidig mappe."
 
-LangString str_msg_rm_run_fail   ${LANG_DANISH}     "Kunne ikke køre afinstallationsguide."
+#LangString str_msg_rm_run_fail   ${LANG_DANISH}     "Kunne ikke køre afinstallationsguide."
 
-LangString str_msg_abort_install ${LANG_DANISH}     "Installationsguiden vil afbryde."
+#LangString str_msg_abort_install ${LANG_DANISH}     "Installationsguiden vil afbryde."
 
 LangString str_msg_install_fail  ${LANG_DANISH}     "Installationen mislykkedes. Bedre held næste gang."
 
 LangString str_msg_rm_exe_fail   ${LANG_DANISH}     "Nogle filer i $0 er ikke blevet slettet!$\r$\nDu skal gøre det manuelt."
 
-LangString str_msg_rm_root_fail  ${LANG_DANISH}     "ADVARSEL: Kan ikke fjerne $\"$vim_install_root$\", den er ikke tom!"
+#LangString str_msg_rm_root_fail  ${LANG_DANISH}     "ADVARSEL: Kan ikke fjerne $\"$vim_install_root$\", den er ikke tom!"
 
 LangString str_msg_wait_uninst   ${LANG_DANISH}     "Venter på at den gamle version bliver afinstalleret..."
 
diff --git a/nsis/lang/dutch.nsi b/nsis/lang/dutch.nsi
index 2aeaef13d..f573da204 100644
--- a/nsis/lang/dutch.nsi
+++ b/nsis/lang/dutch.nsi
@@ -23,8 +23,8 @@ LangString ^UninstallCaption ${LANG_DUTCH} \
 # MUI Configuration Strings                                               {{{1
 ##############################################################################
 
-LangString str_dest_folder          ${LANG_DUTCH} \
-    "Doelmap (moet eindigen op $\"vim$\")"
+#LangString str_dest_folder          ${LANG_DUTCH} \
+#    "Doelmap (moet eindigen op $\"vim$\")"
 
 LangString str_show_readme          ${LANG_DUTCH} \
     "README weergeven na installatie"
@@ -81,27 +81,27 @@ LangString str_desc_start_menu      ${LANG_DUTCH} \
     "Voeg Vim toe aan de programma's map in het start menu.  \
      Van toepassing op Windows 95 en later."
 
-LangString str_section_quick_launch ${LANG_DUTCH} \
-    "In de snel starten balk"
-LangString str_desc_quick_launch    ${LANG_DUTCH} \
-    "Voeg Vim snelkoppeling toe aan de snel starten balk."
+#LangString str_section_quick_launch ${LANG_DUTCH} \
+#    "In de snel starten balk"
+#LangString str_desc_quick_launch    ${LANG_DUTCH} \
+#    "Voeg Vim snelkoppeling toe aan de snel starten balk."
 
 LangString str_section_edit_with    ${LANG_DUTCH} \
     "Voeg Vim contextmenu toe"
 LangString str_desc_edit_with       ${LANG_DUTCH} \
     "Voeg Vim toe aan de $\"Openen met...$\" contextmenu lijst."
 
-LangString str_section_edit_with32  ${LANG_DUTCH} \
-    "32-bit versie"
-LangString str_desc_edit_with32     ${LANG_DUTCH} \
-    "Voeg Vim toe aan de $\"Openen met...$\" contextmenu lijst \
-     voor 32-bit toepassingen."
+#LangString str_section_edit_with32  ${LANG_DUTCH} \
+#    "32-bit versie"
+#LangString str_desc_edit_with32     ${LANG_DUTCH} \
+#    "Voeg Vim toe aan de $\"Openen met...$\" contextmenu lijst \
+#     voor 32-bit toepassingen."
 
-LangString str_section_edit_with64  ${LANG_DUTCH} \
-    "64-bit versie"
-LangString str_desc_edit_with64     ${LANG_DUTCH} \
-    "Voeg Vim toe aan de $\"Openen met...$\" contextmenu lijst \
-     voor 64-bit toepassingen."
+#LangString str_section_edit_with64  ${LANG_DUTCH} \
+#    "64-bit versie"
+#LangString str_desc_edit_with64     ${LANG_DUTCH} \
+#    "Voeg Vim toe aan de $\"Openen met...$\" contextmenu lijst \
+#     voor 64-bit toepassingen."
 
 LangString str_section_vim_rc       ${LANG_DUTCH} \
     "Creëer standaard configuratie"
@@ -161,55 +161,55 @@ LangString str_desc_rm_rootdir      ${LANG_DUTCH} \
 # Messages                                                                {{{1
 ##############################################################################
 
-LangString str_msg_too_many_ver  ${LANG_DUTCH} \
-    "Er zijn $vim_old_ver_count Vim versies op je systeem gevonden.$\r$\n\
-     Deze installatie kan omgaan met maximaal ${VIM_MAX_OLD_VER} versies.$\r$\n\
-     Verwijder a.u.b. wat versies en probeer het dan opnieuw."
+#LangString str_msg_too_many_ver  ${LANG_DUTCH} \
+#    "Er zijn $vim_old_ver_count Vim versies op je systeem gevonden.$\r$\n\
+#     Deze installatie kan omgaan met maximaal ${VIM_MAX_OLD_VER} versies.$\r$\n\
+#     Verwijder a.u.b. wat versies en probeer het dan opnieuw."
 
-LangString str_msg_invalid_root  ${LANG_DUTCH} \
-    "Ongeldig installatiepad: $vim_install_root!$\r$\n\
-     Het moet eindelijk op $\"vim$\"."
+#LangString str_msg_invalid_root  ${LANG_DUTCH} \
+#    "Ongeldig installatiepad: $vim_install_root!$\r$\n\
+#     Het moet eindelijk op $\"vim$\"."
 
-LangString str_msg_bin_mismatch  ${LANG_DUTCH} \
-    "Binair pad onjuist!$\r$\n$\r$\n\
-     Het binaire pad zou $\"$vim_bin_path$\" moeten zijn,$\r$\n\
-     maar het systeem geeft aan dat het binaire pad $\"$INSTDIR$\" is."
+#LangString str_msg_bin_mismatch  ${LANG_DUTCH} \
+#    "Binair pad onjuist!$\r$\n$\r$\n\
+#     Het binaire pad zou $\"$vim_bin_path$\" moeten zijn,$\r$\n\
+#     maar het systeem geeft aan dat het binaire pad $\"$INSTDIR$\" is."
 
-LangString str_msg_vim_running   ${LANG_DUTCH} \
-    "Vim is nog actief op je systeem.$\r$\n\
-     Sluit a.u.b. alle instanties van Vim voordat je verder gaat."
+#LangString str_msg_vim_running   ${LANG_DUTCH} \
+#    "Vim is nog actief op je systeem.$\r$\n\
+#     Sluit a.u.b. alle instanties van Vim voordat je verder gaat."
 
-LangString str_msg_register_ole  ${LANG_DUTCH} \
-    "Bezig met proberen om Vim te registreren met OLE. \
-     Er is geen bericht dat aangeeft of deze operatie slaagt."
+#LangString str_msg_register_ole  ${LANG_DUTCH} \
+#    "Bezig met proberen om Vim te registreren met OLE. \
+#     Er is geen bericht dat aangeeft of deze operatie slaagt."
 
-LangString str_msg_unreg_ole     ${LANG_DUTCH} \
-    "Bezig met proberen om Vim te de-registreren met OLE. \
-     Er is geen bericht dat aangeeft of deze operatie slaagt."
+#LangString str_msg_unreg_ole     ${LANG_DUTCH} \
+#    "Bezig met proberen om Vim te de-registreren met OLE. \
+#     Er is geen bericht dat aangeeft of deze operatie slaagt."
 
-LangString str_msg_rm_start      ${LANG_DUTCH} \
-    "De volgende versies worden verwijderd:"
+#LangString str_msg_rm_start      ${LANG_DUTCH} \
+#    "De volgende versies worden verwijderd:"
 
-LangString str_msg_rm_fail       ${LANG_DUTCH} \
-    "De volgende versies konden niet worden verwijderd:"
+#LangString str_msg_rm_fail       ${LANG_DUTCH} \
+#    "De volgende versies konden niet worden verwijderd:"
 
-LangString str_msg_no_rm_key     ${LANG_DUTCH} \
-    "Kan de uninstaller register sleutel niet vinden."
+#LangString str_msg_no_rm_key     ${LANG_DUTCH} \
+#    "Kan de uninstaller register sleutel niet vinden."
 
-LangString str_msg_no_rm_reg     ${LANG_DUTCH} \
-    "Kan de uninstaller niet vinden via het register."
+#LangString str_msg_no_rm_reg     ${LANG_DUTCH} \
+#    "Kan de uninstaller niet vinden via het register."
 
-LangString str_msg_no_rm_exe     ${LANG_DUTCH} \
-    "Kan geen toegang krijgen tot de uninstaller."
+#LangString str_msg_no_rm_exe     ${LANG_DUTCH} \
+#    "Kan geen toegang krijgen tot de uninstaller."
 
-LangString str_msg_rm_copy_fail  ${LANG_DUTCH} \
-    "Kon de uninstaller niet naar een tijdelijke map kopiëren."
+#LangString str_msg_rm_copy_fail  ${LANG_DUTCH} \
+#    "Kon de uninstaller niet naar een tijdelijke map kopiëren."
 
-LangString str_msg_rm_run_fail   ${LANG_DUTCH} \
-    "Kon de uninstaller niet uitvoeren."
+#LangString str_msg_rm_run_fail   ${LANG_DUTCH} \
+#    "Kon de uninstaller niet uitvoeren."
 
-LangString str_msg_abort_install ${LANG_DUTCH} \
-    "Installatie wordt gestopt."
+#LangString str_msg_abort_install ${LANG_DUTCH} \
+#    "Installatie wordt gestopt."
 
 LangString str_msg_install_fail  ${LANG_DUTCH} \
     "Installatie is mislukt."
@@ -218,8 +218,8 @@ LangString str_msg_rm_exe_fail   ${LANG_DUTCH} \
     "Sommige bestanden in $0 zijn niet verwijderd!$\r$\n\
      Dit moet je handmatig doen."
 
-LangString str_msg_rm_root_fail  ${LANG_DUTCH} \
-    "WAARSCHUWING: Kan $\"$vim_install_root$\" niet verwijderen omdat het niet leeg is!"
+#LangString str_msg_rm_root_fail  ${LANG_DUTCH} \
+#    "WAARSCHUWING: Kan $\"$vim_install_root$\" niet verwijderen omdat het niet leeg is!"
 
 LangString str_msg_wait_uninst   ${LANG_DUTCH} \
     "Waiting the old version to be uninstalled..."
diff --git a/nsis/lang/english.nsi b/nsis/lang/english.nsi
index 3be97653d..4af0dbf0b 100644
--- a/nsis/lang/english.nsi
+++ b/nsis/lang/english.nsi
@@ -32,8 +32,8 @@ LangString ^UninstallCaption ${LANG_ENGLISH} \
 # MUI Configuration Strings                                               {{{1
 ##############################################################################
 
-LangString str_dest_folder          ${LANG_ENGLISH} \
-    "Destination Folder (Must end with $\"vim$\")"
+#LangString str_dest_folder          ${LANG_ENGLISH} \
+#    "Destination Folder (Must end with $\"vim$\")"
 
 LangString str_show_readme          ${LANG_ENGLISH} \
     "Show README after installation finish"
@@ -89,27 +89,27 @@ LangString str_section_start_menu   ${LANG_ENGLISH} \
 LangString str_desc_start_menu      ${LANG_ENGLISH} \
     "Add Vim in the programs folder of the start menu."
 
-LangString str_section_quick_launch ${LANG_ENGLISH} \
-    "In the Quick Launch Bar"
-LangString str_desc_quick_launch    ${LANG_ENGLISH} \
-    "Add Vim shortcut in the quick launch bar."
+#LangString str_section_quick_launch ${LANG_ENGLISH} \
+#    "In the Quick Launch Bar"
+#LangString str_desc_quick_launch    ${LANG_ENGLISH} \
+#    "Add Vim shortcut in the quick launch bar."
 
 LangString str_section_edit_with    ${LANG_ENGLISH} \
     "Add Vim Context Menu"
 LangString str_desc_edit_with       ${LANG_ENGLISH} \
     "Add Vim to the $\"Open With...$\" context menu list."
 
-LangString str_section_edit_with32  ${LANG_ENGLISH} \
-    "32-bit Version"
-LangString str_desc_edit_with32     ${LANG_ENGLISH} \
-    "Add Vim to the $\"Open With...$\" context menu list \
-     for 32-bit applications."
+#LangString str_section_edit_with32  ${LANG_ENGLISH} \
+#    "32-bit Version"
+#LangString str_desc_edit_with32     ${LANG_ENGLISH} \
+#    "Add Vim to the $\"Open With...$\" context menu list \
+#     for 32-bit applications."
 
-LangString str_section_edit_with64  ${LANG_ENGLISH} \
-    "64-bit Version"
-LangString str_desc_edit_with64     ${LANG_ENGLISH} \
-    "Add Vim to the $\"Open With...$\" context menu list \
-     for 64-bit applications."
+#LangString str_section_edit_with64  ${LANG_ENGLISH} \
+#    "64-bit Version"
+#LangString str_desc_edit_with64     ${LANG_ENGLISH} \
+#    "Add Vim to the $\"Open With...$\" context menu list \
+#     for 64-bit applications."
 
 LangString str_section_vim_rc       ${LANG_ENGLISH} \
     "Create Default Config"
@@ -169,56 +169,56 @@ LangString str_desc_rm_rootdir      ${LANG_ENGLISH} \
 # Messages                                                                {{{1
 ##############################################################################
 
-LangString str_msg_too_many_ver  ${LANG_ENGLISH} \
-    "Found $vim_old_ver_count Vim versions on your system.$\r$\n\
-     This installer can only handle ${VIM_MAX_OLD_VER} versions \
-     at most.$\r$\n\
-     Please remove some versions and start again."
+#LangString str_msg_too_many_ver  ${LANG_ENGLISH} \
+#    "Found $vim_old_ver_count Vim versions on your system.$\r$\n\
+#     This installer can only handle ${VIM_MAX_OLD_VER} versions \
+#     at most.$\r$\n\
+#     Please remove some versions and start again."
 
-LangString str_msg_invalid_root  ${LANG_ENGLISH} \
-    "Invalid install path: $vim_install_root!$\r$\n\
-     It should end with $\"vim$\"."
+#LangString str_msg_invalid_root  ${LANG_ENGLISH} \
+#    "Invalid install path: $vim_install_root!$\r$\n\
+#     It should end with $\"vim$\"."
 
-LangString str_msg_bin_mismatch  ${LANG_ENGLISH} \
-    "Binary path mismatch!$\r$\n$\r$\n\
-     Expect the binary path to be $\"$vim_bin_path$\",$\r$\n\
-     but system indicates the binary path is $\"$INSTDIR$\"."
+#LangString str_msg_bin_mismatch  ${LANG_ENGLISH} \
+#    "Binary path mismatch!$\r$\n$\r$\n\
+#     Expect the binary path to be $\"$vim_bin_path$\",$\r$\n\
+#     but system indicates the binary path is $\"$INSTDIR$\"."
 
-LangString str_msg_vim_running   ${LANG_ENGLISH} \
-    "Vim is still running on your system.$\r$\n\
-     Please close all instances of Vim before you continue."
+#LangString str_msg_vim_running   ${LANG_ENGLISH} \
+#    "Vim is still running on your system.$\r$\n\
+#     Please close all instances of Vim before you continue."
 
-LangString str_msg_register_ole  ${LANG_ENGLISH} \
-    "Attempting to register Vim with OLE. \
-     There is no message indicates whether this works or not."
+#LangString str_msg_register_ole  ${LANG_ENGLISH} \
+#    "Attempting to register Vim with OLE. \
+#     There is no message indicates whether this works or not."
 
-LangString str_msg_unreg_ole     ${LANG_ENGLISH} \
-    "Attempting to unregister Vim with OLE. \
-     There is no message indicates whether this works or not."
+#LangString str_msg_unreg_ole     ${LANG_ENGLISH} \
+#    "Attempting to unregister Vim with OLE. \
+#     There is no message indicates whether this works or not."
 
-LangString str_msg_rm_start      ${LANG_ENGLISH} \
-    "Uninstalling the following version:"
+#LangString str_msg_rm_start      ${LANG_ENGLISH} \
+#    "Uninstalling the following version:"
 
-LangString str_msg_rm_fail       ${LANG_ENGLISH} \
-    "Fail to uninstall the following version:"
+#LangString str_msg_rm_fail       ${LANG_ENGLISH} \
+#    "Fail to uninstall the following version:"
 
-LangString str_msg_no_rm_key     ${LANG_ENGLISH} \
-    "Cannot find uninstaller registry key."
+#LangString str_msg_no_rm_key     ${LANG_ENGLISH} \
+#    "Cannot find uninstaller registry key."
 
-LangString str_msg_no_rm_reg     ${LANG_ENGLISH} \
-    "Cannot find uninstaller from registry."
+#LangString str_msg_no_rm_reg     ${LANG_ENGLISH} \
+#    "Cannot find uninstaller from registry."
 
-LangString str_msg_no_rm_exe     ${LANG_ENGLISH} \
-    "Cannot access uninstaller."
+#LangString str_msg_no_rm_exe     ${LANG_ENGLISH} \
+#    "Cannot access uninstaller."
 
-LangString str_msg_rm_copy_fail  ${LANG_ENGLISH} \
-    "Fail to copy uninstaller to temporary directory."
+#LangString str_msg_rm_copy_fail  ${LANG_ENGLISH} \
+#    "Fail to copy uninstaller to temporary directory."
 
-LangString str_msg_rm_run_fail   ${LANG_ENGLISH} \
-    "Fail to run uninstaller."
+#LangString str_msg_rm_run_fail   ${LANG_ENGLISH} \
+#    "Fail to run uninstaller."
 
-LangString str_msg_abort_install ${LANG_ENGLISH} \
-    "Installer will abort."
+#LangString str_msg_abort_install ${LANG_ENGLISH} \
+#    "Installer will abort."
 
 LangString str_msg_install_fail  ${LANG_ENGLISH} \
     "Installation failed. Better luck next time."
@@ -227,8 +227,8 @@ LangString str_msg_rm_exe_fail   ${LANG_ENGLISH} \
     "Some files in $0 have not been deleted!$\r$\n\
      You must do it manually."
 
-LangString str_msg_rm_root_fail  ${LANG_ENGLISH} \
-    "WARNING: Cannot remove $\"$vim_install_root$\", it is not empty!"
+#LangString str_msg_rm_root_fail  ${LANG_ENGLISH} \
+#    "WARNING: Cannot remove $\"$vim_install_root$\", it is not empty!"
 
 LangString str_msg_wait_uninst   ${LANG_ENGLISH} \
     "Waiting the old version to be uninstalled..."
diff --git a/nsis/lang/german.nsi b/nsis/lang/german.nsi
index 48284cc3c..58d929191 100644
--- a/nsis/lang/german.nsi
+++ b/nsis/lang/german.nsi
@@ -22,8 +22,8 @@ LangString ^UninstallCaption ${LANG_GERMAN} \
 # MUI Configuration Strings                                               {{{1
 ##############################################################################
 
-LangString str_dest_folder          ${LANG_GERMAN} \
-    "Zielverzeichnis auswählen (muss auf $\"vim$\" enden)"
+#LangString str_dest_folder          ${LANG_GERMAN} \
+#    "Zielverzeichnis auswählen (muss auf $\"vim$\" enden)"
 
 LangString str_show_readme          ${LANG_GERMAN} \
     "README-Datei nach der Installation anzeigen"
@@ -80,27 +80,27 @@ LangString str_desc_start_menu      ${LANG_GERMAN} \
     "Vim im Programmverzeichnis des Startmenüs hinzufügen\
      Windows 95 und höher."
 
-LangString str_section_quick_launch ${LANG_GERMAN} \
-    "In der Schnellstartleiste"
-LangString str_desc_quick_launch    ${LANG_GERMAN} \
-    "Verknüpfung zu Vim in der Schnellstartleiste ablegen."
+#LangString str_section_quick_launch ${LANG_GERMAN} \
+#    "In der Schnellstartleiste"
+#LangString str_desc_quick_launch    ${LANG_GERMAN} \
+#    "Verknüpfung zu Vim in der Schnellstartleiste ablegen."
 
 LangString str_section_edit_with    ${LANG_GERMAN} \
     "Vim zum Kontextmenü hinzufügen"
 LangString str_desc_edit_with       ${LANG_GERMAN} \
     "Vim in das $\"Öffnen mit...$\"-Kontextmenü einfügen."
 
-LangString str_section_edit_with32  ${LANG_GERMAN} \
-    "32-Bit-Version"
-LangString str_desc_edit_with32     ${LANG_GERMAN} \
-    "Vim in das $\"Öffnen mit...$\"-Kontextmenü \
-     für 32-Bit-Anwendungen integrieren."
+#LangString str_section_edit_with32  ${LANG_GERMAN} \
+#    "32-Bit-Version"
+#LangString str_desc_edit_with32     ${LANG_GERMAN} \
+#    "Vim in das $\"Öffnen mit...$\"-Kontextmenü \
+#     für 32-Bit-Anwendungen integrieren."
 
-LangString str_section_edit_with64  ${LANG_GERMAN} \
-    "64-Bit-Version"
-LangString str_desc_edit_with64     ${LANG_GERMAN} \
-    "Vim in das $\"Öffnen mit...$\"-Kontextmenü \
-     für 64-Bit-Anwendungen integrieren."
+#LangString str_section_edit_with64  ${LANG_GERMAN} \
+#    "64-Bit-Version"
+#LangString str_desc_edit_with64     ${LANG_GERMAN} \
+#    "Vim in das $\"Öffnen mit...$\"-Kontextmenü \
+#     für 64-Bit-Anwendungen integrieren."
 
 LangString str_section_vim_rc       ${LANG_GERMAN} \
     "Standard-Konfigurationsdatei erstellen"
@@ -161,54 +161,54 @@ LangString str_desc_rm_rootdir      ${LANG_GERMAN} \
 # Messages                                                                {{{1
 ##############################################################################
 
-LangString str_msg_too_many_ver  ${LANG_GERMAN} \
-    "$vim_old_ver_count Vim-Versionen auf diesem System gefunden..$\r$\n\
-     Dieser Installer kann maximal ${VIM_MAX_OLD_VER} Versionen \
-     handhaben.$\r$\n\
-     Bitte alte Versionen entfernen und noch einmal probieren."
+#LangString str_msg_too_many_ver  ${LANG_GERMAN} \
+#    "$vim_old_ver_count Vim-Versionen auf diesem System gefunden..$\r$\n\
+#     Dieser Installer kann maximal ${VIM_MAX_OLD_VER} Versionen \
+#     handhaben.$\r$\n\
+#     Bitte alte Versionen entfernen und noch einmal probieren."
 
-LangString str_msg_invalid_root  ${LANG_GERMAN} \
-    "Nicht gültiges Installationsverzeichnis: $vim_install_root!$\r$\n\
-     Der Pfad muss auf $\"vim$\" enden."
+#LangString str_msg_invalid_root  ${LANG_GERMAN} \
+#    "Nicht gültiges Installationsverzeichnis: $vim_install_root!$\r$\n\
+#     Der Pfad muss auf $\"vim$\" enden."
 
-LangString str_msg_bin_mismatch  ${LANG_GERMAN} \
-    "Pfaddiskrepanz!$\r$\n$\r$\n\
-     Erwarte Anwendungsverzeichnis $\"$vim_bin_path$\",$\r$\n\
-     aber fand Anwendungspfad $\"$INSTDIR$\" vor."
+#LangString str_msg_bin_mismatch  ${LANG_GERMAN} \
+#    "Pfaddiskrepanz!$\r$\n$\r$\n\
+#     Erwarte Anwendungsverzeichnis $\"$vim_bin_path$\",$\r$\n\
+#     aber fand Anwendungspfad $\"$INSTDIR$\" vor."
 
-LangString str_msg_vim_running   ${LANG_GERMAN} \
-    "Laufender Vim-Prozess erkannt.$\r$\n\
-     Bitte alle laufenden Vim-Prozesse vor dem Fortfahren beenden."
+#LangString str_msg_vim_running   ${LANG_GERMAN} \
+#    "Laufender Vim-Prozess erkannt.$\r$\n\
+#     Bitte alle laufenden Vim-Prozesse vor dem Fortfahren beenden."
 
-LangString str_msg_register_ole  ${LANG_GERMAN} \
-    "Versuche OLE-Registrierung durchzuführen."
+#LangString str_msg_register_ole  ${LANG_GERMAN} \
+#    "Versuche OLE-Registrierung durchzuführen."
 
-LangString str_msg_unreg_ole     ${LANG_GERMAN} \
-    "Versuche OLE-Registrierung zu löschen."
+#LangString str_msg_unreg_ole     ${LANG_GERMAN} \
+#    "Versuche OLE-Registrierung zu löschen."
 
-LangString str_msg_rm_start      ${LANG_GERMAN} \
-    "Deinstalliere die folgende Version:"
+#LangString str_msg_rm_start      ${LANG_GERMAN} \
+#    "Deinstalliere die folgende Version:"
 
-LangString str_msg_rm_fail       ${LANG_GERMAN} \
-    "Deinstallation der Version fehlgeschlagen:"
+#LangString str_msg_rm_fail       ${LANG_GERMAN} \
+#    "Deinstallation der Version fehlgeschlagen:"
 
-LangString str_msg_no_rm_key     ${LANG_GERMAN} \
-    "Deinstallationsschlüssel in der Registrierungsdatenbank nicht gefunden."
+#LangString str_msg_no_rm_key     ${LANG_GERMAN} \
+#    "Deinstallationsschlüssel in der Registrierungsdatenbank nicht gefunden."
 
-LangString str_msg_no_rm_reg     ${LANG_GERMAN} \
-    "Kein Uninstaller in der Registrierungsdatenbank gefunden."
+#LangString str_msg_no_rm_reg     ${LANG_GERMAN} \
+#    "Kein Uninstaller in der Registrierungsdatenbank gefunden."
 
-LangString str_msg_no_rm_exe     ${LANG_GERMAN} \
-    "Kein Zugriff auf den Uninstaller."
+#LangString str_msg_no_rm_exe     ${LANG_GERMAN} \
+#    "Kein Zugriff auf den Uninstaller."
 
-LangString str_msg_rm_copy_fail  ${LANG_GERMAN} \
-    "Fehler beim Kopieren des Uninstallers in ein temporäres Verzeichnis."
+#LangString str_msg_rm_copy_fail  ${LANG_GERMAN} \
+#    "Fehler beim Kopieren des Uninstallers in ein temporäres Verzeichnis."
 
-LangString str_msg_rm_run_fail   ${LANG_GERMAN} \
-    "Fehler beim Aufruf des Uninstallers."
+#LangString str_msg_rm_run_fail   ${LANG_GERMAN} \
+#    "Fehler beim Aufruf des Uninstallers."
 
-LangString str_msg_abort_install ${LANG_GERMAN} \
-    "Installation wird abgebrochen."
+#LangString str_msg_abort_install ${LANG_GERMAN} \
+#    "Installation wird abgebrochen."
 
 LangString str_msg_install_fail  ${LANG_GERMAN} \
     "Installation fehlerhaft beendet."
@@ -217,9 +217,9 @@ LangString str_msg_rm_exe_fail   ${LANG_GERMAN} \
     "Einige Dateien im Pfad $0 konnten nicht gelöscht werden!$\r$\n\
      Diese Dateien müssen manuell gelöscht werden."
 
-LangString str_msg_rm_root_fail  ${LANG_GERMAN} \
-    "Achtung: Kann Verzeichnis $\"$vim_install_root$\" nicht entfernen, \
-     weil es nicht leer ist!"
+#LangString str_msg_rm_root_fail  ${LANG_GERMAN} \
+#    "Achtung: Kann Verzeichnis $\"$vim_install_root$\" nicht entfernen, \
+#     weil es nicht leer ist!"
 
 LangString str_msg_wait_uninst   ${LANG_GERMAN} \
     "Waiting the old version to be uninstalled..."
diff --git a/nsis/lang/italian.nsi b/nsis/lang/italian.nsi
index e32b69e24..eb92c7b53 100644
--- a/nsis/lang/italian.nsi
+++ b/nsis/lang/italian.nsi
@@ -23,8 +23,8 @@ LangString ^UninstallCaption ${LANG_ITALIAN} \
 # MUI Configuration Strings                                               {{{1
 ##############################################################################
 
-LangString str_dest_folder          ${LANG_ITALIAN} \
-    "Cartella d'installazione (il nome deve finire con $\"vim$\")"
+#LangString str_dest_folder          ${LANG_ITALIAN} \
+#    "Cartella d'installazione (il nome deve finire con $\"vim$\")"
 
 LangString str_show_readme          ${LANG_ITALIAN} \
     "Visualizza README al termine dell'installazione"
@@ -81,27 +81,27 @@ LangString str_desc_start_menu      ${LANG_ITALIAN} \
     "Aggiungi Vim alle cartelle del menù START.  \
      Disponibile solo da Windows 95 in avanti."
 
-LangString str_section_quick_launch ${LANG_ITALIAN} \
-    "Nella barra di Avvio Veloce"
-LangString str_desc_quick_launch    ${LANG_ITALIAN} \
-    "Aggiungi un puntatore a Vim nella barra di Avvio Veloce."
+#LangString str_section_quick_launch ${LANG_ITALIAN} \
+#    "Nella barra di Avvio Veloce"
+#LangString str_desc_quick_launch    ${LANG_ITALIAN} \
+#    "Aggiungi un puntatore a Vim nella barra di Avvio Veloce."
 
 LangString str_section_edit_with    ${LANG_ITALIAN} \
     "Aggiungi Vim al Menù Contestuale"
 LangString str_desc_edit_with       ${LANG_ITALIAN} \
     "Aggiungi Vim alla lista contestuale $\"Apri con...$\"."
 
-LangString str_section_edit_with32  ${LANG_ITALIAN} \
-    "Versione a 32-bit"
-LangString str_desc_edit_with32     ${LANG_ITALIAN} \
-    "Aggiungi Vim alla lista contestuale $\"Apri con...$\" \
-     per applicazioni a 32-bit."
+#LangString str_section_edit_with32  ${LANG_ITALIAN} \
+#    "Versione a 32-bit"
+#LangString str_desc_edit_with32     ${LANG_ITALIAN} \
+#    "Aggiungi Vim alla lista contestuale $\"Apri con...$\" \
+#     per applicazioni a 32-bit."
 
-LangString str_section_edit_with64  ${LANG_ITALIAN} \
-    "Versione a 64-bit"
-LangString str_desc_edit_with64     ${LANG_ITALIAN} \
-    "Aggiungi Vim alla lista contestuale $\"Apri con...$\" \
-     per applicazioni a 64-bit."
+#LangString str_section_edit_with64  ${LANG_ITALIAN} \
+#    "Versione a 64-bit"
+#LangString str_desc_edit_with64     ${LANG_ITALIAN} \
+#    "Aggiungi Vim alla lista contestuale $\"Apri con...$\" \
+#     per applicazioni a 64-bit."
 
 LangString str_section_vim_rc       ${LANG_ITALIAN} \
     "Crea Configurazione di default"
@@ -168,57 +168,57 @@ LangString str_desc_rm_rootdir      ${LANG_ITALIAN} \
 # Messages                                                                {{{1
 ##############################################################################
 
-LangString str_msg_too_many_ver  ${LANG_ITALIAN} \
-    "Trovate $vim_old_ver_count versioni di Vim sul vostro sistema.$\r$\n\
-     Questo programma di installazione può gestirne solo \
-     ${VIM_MAX_OLD_VER}.$\r$\n\
-     Disinstallate qualche versione precedente e ricominciate."
+#LangString str_msg_too_many_ver  ${LANG_ITALIAN} \
+#    "Trovate $vim_old_ver_count versioni di Vim sul vostro sistema.$\r$\n\
+#     Questo programma di installazione può gestirne solo \
+#     ${VIM_MAX_OLD_VER}.$\r$\n\
+#     Disinstallate qualche versione precedente e ricominciate."
 
-LangString str_msg_invalid_root  ${LANG_ITALIAN} \
-    "Nome di directory di installazione non valida: $vim_install_root!$\r$\n\
-     Dovrebbe terminare con $\"vim$\"."
+#LangString str_msg_invalid_root  ${LANG_ITALIAN} \
+#    "Nome di directory di installazione non valida: $vim_install_root!$\r$\n\
+#     Dovrebbe terminare con $\"vim$\"."
 
-LangString str_msg_bin_mismatch  ${LANG_ITALIAN} \
-    "Incongruenza di installazione!$\r$\n$\r$\n\
-     Cartella di installazione dev'essere $\"$vim_bin_path$\",$\r$\n\
-     ma il sistema segnala invece $\"$INSTDIR$\"."
+#LangString str_msg_bin_mismatch  ${LANG_ITALIAN} \
+#    "Incongruenza di installazione!$\r$\n$\r$\n\
+#     Cartella di installazione dev'essere $\"$vim_bin_path$\",$\r$\n\
+#     ma il sistema segnala invece $\"$INSTDIR$\"."
 
-LangString str_msg_vim_running   ${LANG_ITALIAN} \
-    "Vim ancora in esecuzione sul vostro sistema.$\r$\n\
-     Chiudete tutte le sessioni attive di Vim per continuare."
+#LangString str_msg_vim_running   ${LANG_ITALIAN} \
+#    "Vim ancora in esecuzione sul vostro sistema.$\r$\n\
+#     Chiudete tutte le sessioni attive di Vim per continuare."
 
-LangString str_msg_register_ole  ${LANG_ITALIAN} \
-    "Tentativo di registrazione di Vim con OLE. \
-     Non ci sono messaggi che indicano se ha funzionato o no."
+#LangString str_msg_register_ole  ${LANG_ITALIAN} \
+#    "Tentativo di registrazione di Vim con OLE. \
+#     Non ci sono messaggi che indicano se ha funzionato o no."
 
-LangString str_msg_unreg_ole     ${LANG_ITALIAN} \
-    "Tentativo di togliere da Registry  Vim con OLE. \
-     Non ci sono messaggi che indicano se ha funzionato o no."
+#LangString str_msg_unreg_ole     ${LANG_ITALIAN} \
+#    "Tentativo di togliere da Registry  Vim con OLE. \
+#     Non ci sono messaggi che indicano se ha funzionato o no."
 
-LangString str_msg_rm_start      ${LANG_ITALIAN} \
-    "Disinstallazione delle seguenti versioni:"
+#LangString str_msg_rm_start      ${LANG_ITALIAN} \
+#    "Disinstallazione delle seguenti versioni:"
 
-LangString str_msg_rm_fail       ${LANG_ITALIAN} \
-    "Disinstallazione non riuscita per la seguente versione:"
+#LangString str_msg_rm_fail       ${LANG_ITALIAN} \
+#    "Disinstallazione non riuscita per la seguente versione:"
 
-LangString str_msg_no_rm_key     ${LANG_ITALIAN} \
-    "Non riesco a trovare chiave di disinstallazione nel Registry."
+#LangString str_msg_no_rm_key     ${LANG_ITALIAN} \
+#    "Non riesco a trovare chiave di disinstallazione nel Registry."
 
-LangString str_msg_no_rm_reg     ${LANG_ITALIAN} \
-    "Non riesco a trovare programma disinstallazione nel Registry."
+#LangString str_msg_no_rm_reg     ${LANG_ITALIAN} \
+#    "Non riesco a trovare programma disinstallazione nel Registry."
 
-LangString str_msg_no_rm_exe     ${LANG_ITALIAN} \
-    "Non riesco a utilizzare programma disinstallazione."
+#LangString str_msg_no_rm_exe     ${LANG_ITALIAN} \
+#    "Non riesco a utilizzare programma disinstallazione."
 
-LangString str_msg_rm_copy_fail  ${LANG_ITALIAN} \
-    "Non riesco a copiare programma disinstallazione a una \
-     directory temporanea."
+#LangString str_msg_rm_copy_fail  ${LANG_ITALIAN} \
+#    "Non riesco a copiare programma disinstallazione a una \
+#     directory temporanea."
 
-LangString str_msg_rm_run_fail   ${LANG_ITALIAN} \
-    "Non riesco a eseguire programma disinstallazione."
+#LangString str_msg_rm_run_fail   ${LANG_ITALIAN} \
+#    "Non riesco a eseguire programma disinstallazione."
 
-LangString str_msg_abort_install ${LANG_ITALIAN} \
-    "Il programma di disinstallazione verrà chiuso senza aver fatto nulla."
+#LangString str_msg_abort_install ${LANG_ITALIAN} \
+#    "Il programma di disinstallazione verrà chiuso senza aver fatto nulla."
 
 LangString str_msg_install_fail  ${LANG_ITALIAN} \
     "Installazione non riuscita. Miglior fortuna alla prossima!"
@@ -227,8 +227,8 @@ LangString str_msg_rm_exe_fail   ${LANG_ITALIAN} \
     "Alcuni file in $0 non sono stati cancellati!$\r$\n\
      Dovreste cancellarli voi stessi."
 
-LangString str_msg_rm_root_fail  ${LANG_ITALIAN} \
-    "AVVISO: Non posso cancellare $\"$vim_install_root$\", non è vuota!"
+#LangString str_msg_rm_root_fail  ${LANG_ITALIAN} \
+#    "AVVISO: Non posso cancellare $\"$vim_install_root$\", non è vuota!"
 
 LangString str_msg_wait_uninst   ${LANG_ITALIAN} \
     "Waiting the old version to be uninstalled..."
diff --git a/nsis/lang/japanese.nsi b/nsis/lang/japanese.nsi
index a6a915512..644f4e5c3 100644
--- a/nsis/lang/japanese.nsi
+++ b/nsis/lang/japanese.nsi
@@ -88,27 +88,27 @@ LangString str_section_start_menu   ${LANG_JAPANESE} \
 LangString str_desc_start_menu      ${LANG_JAPANESE} \
     "Vim のアイコンをスタートメニューのプログラムフォルダー上に作成します。"
 
-LangString str_section_quick_launch ${LANG_JAPANESE} \
-    "In the Quick Launch Bar"
-LangString str_desc_quick_launch    ${LANG_JAPANESE} \
-    "Add Vim shortcut in the quick launch bar."
+#LangString str_section_quick_launch ${LANG_JAPANESE} \
+#    "In the Quick Launch Bar"
+#LangString str_desc_quick_launch    ${LANG_JAPANESE} \
+#    "Add Vim shortcut in the quick launch bar."
 
 LangString str_section_edit_with    ${LANG_JAPANESE} \
     "Vim のコンテキストメニューを追加"
 LangString str_desc_edit_with       ${LANG_JAPANESE} \
     "$\"Vimで編集する$\" をコンテキストメニューに追加します。"
 
-LangString str_section_edit_with32  ${LANG_JAPANESE} \
-    "32-bit Version"
-LangString str_desc_edit_with32     ${LANG_JAPANESE} \
-    "Add Vim to the $\"Open With...$\" context menu list \
-     for 32-bit applications."
+#LangString str_section_edit_with32  ${LANG_JAPANESE} \
+#    "32-bit Version"
+#LangString str_desc_edit_with32     ${LANG_JAPANESE} \
+#    "Add Vim to the $\"Open With...$\" context menu list \
+#     for 32-bit applications."
 
-LangString str_section_edit_with64  ${LANG_JAPANESE} \
-    "64-bit Version"
-LangString str_desc_edit_with64     ${LANG_JAPANESE} \
-    "Add Vim to the $\"Open With...$\" context menu list \
-     for 64-bit applications."
+#LangString str_section_edit_with64  ${LANG_JAPANESE} \
+#    "64-bit Version"
+#LangString str_desc_edit_with64     ${LANG_JAPANESE} \
+#    "Add Vim to the $\"Open With...$\" context menu list \
+#     for 64-bit applications."
 
 LangString str_section_vim_rc       ${LANG_JAPANESE} \
     "既定のコンフィグを作成"
@@ -165,56 +165,56 @@ LangString str_desc_rm_rootdir      ${LANG_JAPANESE} \
 # Messages                                                                {{{1
 ##############################################################################
 
-LangString str_msg_too_many_ver  ${LANG_JAPANESE} \
-    "Found $vim_old_ver_count Vim versions on your system.$\r$\n\
-     This installer can only handle ${VIM_MAX_OLD_VER} versions \
-     at most.$\r$\n\
-     Please remove some versions and start again."
+#LangString str_msg_too_many_ver  ${LANG_JAPANESE} \
+#    "Found $vim_old_ver_count Vim versions on your system.$\r$\n\
+#     This installer can only handle ${VIM_MAX_OLD_VER} versions \
+#     at most.$\r$\n\
+#     Please remove some versions and start again."
 
-LangString str_msg_invalid_root  ${LANG_JAPANESE} \
-    "Invalid install path: $vim_install_root!$\r$\n\
-     It should end with $\"vim$\"."
+#LangString str_msg_invalid_root  ${LANG_JAPANESE} \
+#    "Invalid install path: $vim_install_root!$\r$\n\
+#     It should end with $\"vim$\"."
 
-LangString str_msg_bin_mismatch  ${LANG_JAPANESE} \
-    "Binary path mismatch!$\r$\n$\r$\n\
-     Expect the binary path to be $\"$vim_bin_path$\",$\r$\n\
-     but system indicates the binary path is $\"$INSTDIR$\"."
+#LangString str_msg_bin_mismatch  ${LANG_JAPANESE} \
+#    "Binary path mismatch!$\r$\n$\r$\n\
+#     Expect the binary path to be $\"$vim_bin_path$\",$\r$\n\
+#     but system indicates the binary path is $\"$INSTDIR$\"."
 
-LangString str_msg_vim_running   ${LANG_JAPANESE} \
-    "Vim is still running on your system.$\r$\n\
-     Please close all instances of Vim before you continue."
+#LangString str_msg_vim_running   ${LANG_JAPANESE} \
+#    "Vim is still running on your system.$\r$\n\
+#     Please close all instances of Vim before you continue."
 
-LangString str_msg_register_ole  ${LANG_JAPANESE} \
-    "Attempting to register Vim with OLE. \
-     There is no message indicates whether this works or not."
+#LangString str_msg_register_ole  ${LANG_JAPANESE} \
+#    "Attempting to register Vim with OLE. \
+#     There is no message indicates whether this works or not."
 
-LangString str_msg_unreg_ole     ${LANG_JAPANESE} \
-    "Attempting to unregister Vim with OLE. \
-     There is no message indicates whether this works or not."
+#LangString str_msg_unreg_ole     ${LANG_JAPANESE} \
+#    "Attempting to unregister Vim with OLE. \
+#     There is no message indicates whether this works or not."
 
-LangString str_msg_rm_start      ${LANG_JAPANESE} \
-    "Uninstalling the following version:"
+#LangString str_msg_rm_start      ${LANG_JAPANESE} \
+#    "Uninstalling the following version:"
 
-LangString str_msg_rm_fail       ${LANG_JAPANESE} \
-    "Fail to uninstall the following version:"
+#LangString str_msg_rm_fail       ${LANG_JAPANESE} \
+#    "Fail to uninstall the following version:"
 
-LangString str_msg_no_rm_key     ${LANG_JAPANESE} \
-    "Cannot find uninstaller registry key."
+#LangString str_msg_no_rm_key     ${LANG_JAPANESE} \
+#    "Cannot find uninstaller registry key."
 
-LangString str_msg_no_rm_reg     ${LANG_JAPANESE} \
-    "Cannot find uninstaller from registry."
+#LangString str_msg_no_rm_reg     ${LANG_JAPANESE} \
+#    "Cannot find uninstaller from registry."
 
-LangString str_msg_no_rm_exe     ${LANG_JAPANESE} \
-    "Cannot access uninstaller."
+#LangString str_msg_no_rm_exe     ${LANG_JAPANESE} \
+#    "Cannot access uninstaller."
 
-LangString str_msg_rm_copy_fail  ${LANG_JAPANESE} \
-    "Fail to copy uninstaller to temporary directory."
+#LangString str_msg_rm_copy_fail  ${LANG_JAPANESE} \
+#    "Fail to copy uninstaller to temporary directory."
 
-LangString str_msg_rm_run_fail   ${LANG_JAPANESE} \
-    "Fail to run uninstaller."
+#LangString str_msg_rm_run_fail   ${LANG_JAPANESE} \
+#    "Fail to run uninstaller."
 
-LangString str_msg_abort_install ${LANG_JAPANESE} \
-    "Installer will abort."
+#LangString str_msg_abort_install ${LANG_JAPANESE} \
+#    "Installer will abort."
 
 LangString str_msg_install_fail  ${LANG_JAPANESE} \
     "インストールに失敗しました。次はうまくいくことを祈ります。"
@@ -223,8 +223,8 @@ LangString str_msg_rm_exe_fail   ${LANG_JAPANESE} \
     "$0 内の一部のファイルは削除できませんでした!$\r$\n\
      手動で削除する必要があります。"
 
-LangString str_msg_rm_root_fail  ${LANG_JAPANESE} \
-    "WARNING: Cannot remove $\"$vim_install_root$\", it is not empty!"
+#LangString str_msg_rm_root_fail  ${LANG_JAPANESE} \
+#    "WARNING: Cannot remove $\"$vim_install_root$\", it is not empty!"
 
 LangString str_msg_wait_uninst   ${LANG_JAPANESE} \
     "古いバージョンがアンインストールされるのを待っています..."
diff --git a/nsis/lang/simpchinese.nsi b/nsis/lang/simpchinese.nsi
index c4b2ea2d9..e071d8311 100644
--- a/nsis/lang/simpchinese.nsi
+++ b/nsis/lang/simpchinese.nsi
@@ -23,8 +23,8 @@ LangString ^UninstallCaption ${LANG_SIMPCHINESE} \
 # MUI Configuration Strings                                               {{{1
 ##############################################################################
 
-LangString str_dest_folder          ${LANG_SIMPCHINESE} \
-    "安装路径 (必须以 vim 结尾)"
+#LangString str_dest_folder          ${LANG_SIMPCHINESE} \
+#    "安装路径 (必须以 vim 结尾)"
 
 LangString str_show_readme          ${LANG_SIMPCHINESE} \
     "安装完成后显示 README 文件"
@@ -79,25 +79,25 @@ LangString str_section_start_menu   ${LANG_SIMPCHINESE} \
 LangString str_desc_start_menu      ${LANG_SIMPCHINESE} \
     "在启动菜单的程序菜单下添加 Vim 组。适用于 Windows 95 及以上版本。"
 
-LangString str_section_quick_launch ${LANG_SIMPCHINESE} \
-    "在快速启动启动栏中"
-LangString str_desc_quick_launch    ${LANG_SIMPCHINESE} \
-    "在快速启动栏中添加 Vim 图标。"
+#LangString str_section_quick_launch ${LANG_SIMPCHINESE} \
+#    "在快速启动启动栏中"
+#LangString str_desc_quick_launch    ${LANG_SIMPCHINESE} \
+#    "在快速启动栏中添加 Vim 图标。"
 
 LangString str_section_edit_with    ${LANG_SIMPCHINESE} \
     "安装快捷菜单"
 LangString str_desc_edit_with       ${LANG_SIMPCHINESE} \
     "将 Vim 添加到“打开方式”快捷菜单中。"
 
-LangString str_section_edit_with32  ${LANG_SIMPCHINESE} \
-    "32 位版本"
-LangString str_desc_edit_with32     ${LANG_SIMPCHINESE} \
-    "将 Vim 添加到 32 位程序的“打开方式”快捷菜单中。"
+#LangString str_section_edit_with32  ${LANG_SIMPCHINESE} \
+#    "32 位版本"
+#LangString str_desc_edit_with32     ${LANG_SIMPCHINESE} \
+#    "将 Vim 添加到 32 位程序的“打开方式”快捷菜单中。"
 
-LangString str_section_edit_with64  ${LANG_SIMPCHINESE} \
-    "64 位版本"
-LangString str_desc_edit_with64     ${LANG_SIMPCHINESE} \
-    "将 Vim 添加到 64 位程序的“打开方式”快捷菜单中。"
+#LangString str_section_edit_with64  ${LANG_SIMPCHINESE} \
+#    "64 位版本"
+#LangString str_desc_edit_with64     ${LANG_SIMPCHINESE} \
+#    "将 Vim 添加到 64 位程序的“打开方式”快捷菜单中。"
 
 LangString str_section_vim_rc       ${LANG_SIMPCHINESE} \
     "创建缺省配置文件"
@@ -158,53 +158,53 @@ LangString str_desc_rm_rootdir      ${LANG_SIMPCHINESE} \
 # Messages                                                                {{{1
 ##############################################################################
 
-LangString str_msg_too_many_ver  ${LANG_SIMPCHINESE} \
-    "您的系统上安装了 $vim_old_ver_count 个不同版本的 Vim，$\r$\n\
-     但本安装程序最多只能处理 ${VIM_MAX_OLD_VER} 个版本。$\r$\n\
-     请您手工删除一些旧版本以后再运行本安装程序。"
+#LangString str_msg_too_many_ver  ${LANG_SIMPCHINESE} \
+#    "您的系统上安装了 $vim_old_ver_count 个不同版本的 Vim，$\r$\n\
+#     但本安装程序最多只能处理 ${VIM_MAX_OLD_VER} 个版本。$\r$\n\
+#     请您手工删除一些旧版本以后再运行本安装程序。"
 
-LangString str_msg_invalid_root  ${LANG_SIMPCHINESE} \
-    "安装路径“$vim_install_root”无效！$\r$\n\
-     该路径必须以 vim 结尾。"
+#LangString str_msg_invalid_root  ${LANG_SIMPCHINESE} \
+#    "安装路径“$vim_install_root”无效！$\r$\n\
+#     该路径必须以 vim 结尾。"
 
-LangString str_msg_bin_mismatch  ${LANG_SIMPCHINESE} \
-    "Vim 执行程序安装路径异常！$\r$\n$\r$\n\
-     该版本 Vim 的执行程序安装路径应该是“$vim_bin_path”,$\r$\n\
-     而系统却指示该路径为“$INSTDIR”。"
+#LangString str_msg_bin_mismatch  ${LANG_SIMPCHINESE} \
+#    "Vim 执行程序安装路径异常！$\r$\n$\r$\n\
+#     该版本 Vim 的执行程序安装路径应该是“$vim_bin_path”,$\r$\n\
+#     而系统却指示该路径为“$INSTDIR”。"
 
-LangString str_msg_vim_running   ${LANG_SIMPCHINESE} \
-    "您的系统上仍有 Vim 在运行，$\r$\n\
-     请您在执行后续步骤前退出这些 Vim。"
+#LangString str_msg_vim_running   ${LANG_SIMPCHINESE} \
+#    "您的系统上仍有 Vim 在运行，$\r$\n\
+#     请您在执行后续步骤前退出这些 Vim。"
 
-LangString str_msg_register_ole  ${LANG_SIMPCHINESE} \
-    "试图注册 Vim OLE 服务器。请注意无论成功与否都不再显示进一步的信息。"
+#LangString str_msg_register_ole  ${LANG_SIMPCHINESE} \
+#    "试图注册 Vim OLE 服务器。请注意无论成功与否都不再显示进一步的信息。"
 
-LangString str_msg_unreg_ole     ${LANG_SIMPCHINESE} \
-    "试图注销 Vim OLE 服务器。请注意无论成功与否都不再显示进一步的信息。"
+#LangString str_msg_unreg_ole     ${LANG_SIMPCHINESE} \
+#    "试图注销 Vim OLE 服务器。请注意无论成功与否都不再显示进一步的信息。"
 
-LangString str_msg_rm_start      ${LANG_SIMPCHINESE} \
-    "开始卸载以下版本："
+#LangString str_msg_rm_start      ${LANG_SIMPCHINESE} \
+#    "开始卸载以下版本："
 
-LangString str_msg_rm_fail       ${LANG_SIMPCHINESE} \
-    "以下版本卸载失败："
+#LangString str_msg_rm_fail       ${LANG_SIMPCHINESE} \
+#    "以下版本卸载失败："
 
-LangString str_msg_no_rm_key     ${LANG_SIMPCHINESE} \
-    "找不到卸载程序的注册表键。"
+#LangString str_msg_no_rm_key     ${LANG_SIMPCHINESE} \
+#    "找不到卸载程序的注册表键。"
 
-LangString str_msg_no_rm_reg     ${LANG_SIMPCHINESE} \
-    "在注册表中未找到卸载程序路径。"
+#LangString str_msg_no_rm_reg     ${LANG_SIMPCHINESE} \
+#    "在注册表中未找到卸载程序路径。"
 
-LangString str_msg_no_rm_exe     ${LANG_SIMPCHINESE} \
-    "找不到卸载程序。"
+#LangString str_msg_no_rm_exe     ${LANG_SIMPCHINESE} \
+#    "找不到卸载程序。"
 
-LangString str_msg_rm_copy_fail  ${LANG_SIMPCHINESE} \
-    "无法将卸载程序复制到临时目录。"
+#LangString str_msg_rm_copy_fail  ${LANG_SIMPCHINESE} \
+#    "无法将卸载程序复制到临时目录。"
 
-LangString str_msg_rm_run_fail   ${LANG_SIMPCHINESE} \
-    "执行卸载程序失败。"
+#LangString str_msg_rm_run_fail   ${LANG_SIMPCHINESE} \
+#    "执行卸载程序失败。"
 
-LangString str_msg_abort_install ${LANG_SIMPCHINESE} \
-    "安装程序将退出。"
+#LangString str_msg_abort_install ${LANG_SIMPCHINESE} \
+#    "安装程序将退出。"
 
 LangString str_msg_install_fail  ${LANG_SIMPCHINESE} \
     "安装失败。祝您下次好运。"
@@ -213,9 +213,9 @@ LangString str_msg_rm_exe_fail   ${LANG_SIMPCHINESE} \
     "目录“$0”下有部分文件删除失败！$\r$\n\
      您只能手工删除该目录。"
 
-LangString str_msg_rm_root_fail  ${LANG_SIMPCHINESE} \
-    "警告：无法删除 Vim 安装目录“$vim_install_root”，\
-     该目录下仍有其他文件。"
+#LangString str_msg_rm_root_fail  ${LANG_SIMPCHINESE} \
+#    "警告：无法删除 Vim 安装目录“$vim_install_root”，\
+#     该目录下仍有其他文件。"
 
 LangString str_msg_wait_uninst   ${LANG_SIMPCHINESE} \
     "Waiting the old version to be uninstalled..."
diff --git a/nsis/lang/tradchinese.nsi b/nsis/lang/tradchinese.nsi
index d65ba659b..1394745ac 100644
--- a/nsis/lang/tradchinese.nsi
+++ b/nsis/lang/tradchinese.nsi
@@ -23,8 +23,8 @@ LangString ^UninstallCaption ${LANG_TRADCHINESE} \
 # MUI Configuration Strings                                               {{{1
 ##############################################################################
 
-LangString str_dest_folder          ${LANG_TRADCHINESE} \
-    "安裝資料夾 (必須以 vim 結尾)"
+#LangString str_dest_folder          ${LANG_TRADCHINESE} \
+#    "安裝資料夾 (必須以 vim 結尾)"
 
 LangString str_show_readme          ${LANG_TRADCHINESE} \
     "安裝完成後顯示 README 檔案"
@@ -80,25 +80,25 @@ LangString str_desc_start_menu      ${LANG_TRADCHINESE} \
     "在「開始」功能表的「程式」集中建立 Vim 啟動組。\
      適用于 Windows 95 及以上版本。"
 
-LangString str_section_quick_launch ${LANG_TRADCHINESE} \
-    "於快速啟動列"
-LangString str_desc_quick_launch    ${LANG_TRADCHINESE} \
-    "在快速啟動列中建立 Vim 圖示。"
+#LangString str_section_quick_launch ${LANG_TRADCHINESE} \
+#    "於快速啟動列"
+#LangString str_desc_quick_launch    ${LANG_TRADCHINESE} \
+#    "在快速啟動列中建立 Vim 圖示。"
 
 LangString str_section_edit_with    ${LANG_TRADCHINESE} \
     "安裝快捷選單"
 LangString str_desc_edit_with       ${LANG_TRADCHINESE} \
     "在「打開方式」快捷選單中添加 Vim 項。"
 
-LangString str_section_edit_with32  ${LANG_TRADCHINESE} \
-    "32 位元版本"
-LangString str_desc_edit_with32     ${LANG_TRADCHINESE} \
-    "在 32 位元程式的「打開方式」快捷選單中添加 Vim 項。"
+#LangString str_section_edit_with32  ${LANG_TRADCHINESE} \
+#    "32 位元版本"
+#LangString str_desc_edit_with32     ${LANG_TRADCHINESE} \
+#    "在 32 位元程式的「打開方式」快捷選單中添加 Vim 項。"
 
-LangString str_section_edit_with64  ${LANG_TRADCHINESE} \
-    "64 位元版本"
-LangString str_desc_edit_with64     ${LANG_TRADCHINESE} \
-    "在 64 位元程式的「打開方式」快捷選單中添加 Vim 項。"
+#LangString str_section_edit_with64  ${LANG_TRADCHINESE} \
+#    "64 位元版本"
+#LangString str_desc_edit_with64     ${LANG_TRADCHINESE} \
+#    "在 64 位元程式的「打開方式」快捷選單中添加 Vim 項。"
 
 LangString str_section_vim_rc       ${LANG_TRADCHINESE} \
     "建立默認設定檔"
@@ -159,53 +159,53 @@ LangString str_desc_rm_rootdir      ${LANG_TRADCHINESE} \
 # Messages                                                                {{{1
 ##############################################################################
 
-LangString str_msg_too_many_ver  ${LANG_TRADCHINESE} \
-    "閣下的電腦上安裝了 $vim_old_ver_count 個不同版本的 Vim，$\r$\n\
-     但是本安裝程式最多只能處理 ${VIM_MAX_OLD_VER} 個版本。$\r$\n\
-     煩請閣下手工移除一些版本以后再運行本安裝程式。"
+#LangString str_msg_too_many_ver  ${LANG_TRADCHINESE} \
+#    "閣下的電腦上安裝了 $vim_old_ver_count 個不同版本的 Vim，$\r$\n\
+#     但是本安裝程式最多只能處理 ${VIM_MAX_OLD_VER} 個版本。$\r$\n\
+#     煩請閣下手工移除一些版本以后再運行本安裝程式。"
 
-LangString str_msg_invalid_root  ${LANG_TRADCHINESE} \
-    "安裝資料夾「$vim_install_root」無效！$\r$\n\
-     該資料夾必須以「vim」結尾。"
+#LangString str_msg_invalid_root  ${LANG_TRADCHINESE} \
+#    "安裝資料夾「$vim_install_root」無效！$\r$\n\
+#     該資料夾必須以「vim」結尾。"
 
-LangString str_msg_bin_mismatch  ${LANG_TRADCHINESE} \
-    "Vim 執行程式安裝路徑異常！$\r$\n$\r$\n\
-     該版本 Vim 的執行程式安裝路徑應該是「$vim_bin_path」,$\r$\n\
-     而系統卻指示該路徑為「$INSTDIR」。"
+#LangString str_msg_bin_mismatch  ${LANG_TRADCHINESE} \
+#    "Vim 執行程式安裝路徑異常！$\r$\n$\r$\n\
+#     該版本 Vim 的執行程式安裝路徑應該是「$vim_bin_path」,$\r$\n\
+#     而系統卻指示該路徑為「$INSTDIR」。"
 
-LangString str_msg_vim_running   ${LANG_TRADCHINESE} \
-    "閣下的電腦上尚有正在運行之 Vim，$\r$\n\
-     煩請閣下在執行后續步驟前將其全部退出。"
+#LangString str_msg_vim_running   ${LANG_TRADCHINESE} \
+#    "閣下的電腦上尚有正在運行之 Vim，$\r$\n\
+#     煩請閣下在執行后續步驟前將其全部退出。"
 
-LangString str_msg_register_ole  ${LANG_TRADCHINESE} \
-    "試圖注冊 Vim OLE 伺服程式。請注意不論成功與否都不再顯示進一步的信息。"
+#LangString str_msg_register_ole  ${LANG_TRADCHINESE} \
+#    "試圖注冊 Vim OLE 伺服程式。請注意不論成功與否都不再顯示進一步的信息。"
 
-LangString str_msg_unreg_ole     ${LANG_TRADCHINESE} \
-    "試圖注銷 Vim OLE 伺服程式。請注意不論成功與否都不再顯示進一步的信息。"
+#LangString str_msg_unreg_ole     ${LANG_TRADCHINESE} \
+#    "試圖注銷 Vim OLE 伺服程式。請注意不論成功與否都不再顯示進一步的信息。"
 
-LangString str_msg_rm_start      ${LANG_TRADCHINESE} \
-    "正移除如下版本："
+#LangString str_msg_rm_start      ${LANG_TRADCHINESE} \
+#    "正移除如下版本："
 
-LangString str_msg_rm_fail       ${LANG_TRADCHINESE} \
-    "以下版本移除失敗："
+#LangString str_msg_rm_fail       ${LANG_TRADCHINESE} \
+#    "以下版本移除失敗："
 
-LangString str_msg_no_rm_key     ${LANG_TRADCHINESE} \
-    "找不到反安裝程式的登錄檔入口。"
+#LangString str_msg_no_rm_key     ${LANG_TRADCHINESE} \
+#    "找不到反安裝程式的登錄檔入口。"
 
-LangString str_msg_no_rm_reg     ${LANG_TRADCHINESE} \
-    "在登錄檔中未找到反安裝程式路徑。"
+#LangString str_msg_no_rm_reg     ${LANG_TRADCHINESE} \
+#    "在登錄檔中未找到反安裝程式路徑。"
 
-LangString str_msg_no_rm_exe     ${LANG_TRADCHINESE} \
-    "找不到反安裝程式。"
+#LangString str_msg_no_rm_exe     ${LANG_TRADCHINESE} \
+#    "找不到反安裝程式。"
 
-LangString str_msg_rm_copy_fail  ${LANG_TRADCHINESE} \
-    "無法將法將反安裝程式复制到臨時目錄。"
+#LangString str_msg_rm_copy_fail  ${LANG_TRADCHINESE} \
+#    "無法將法將反安裝程式复制到臨時目錄。"
 
-LangString str_msg_rm_run_fail   ${LANG_TRADCHINESE} \
-    "執行反安裝程式失敗。"
+#LangString str_msg_rm_run_fail   ${LANG_TRADCHINESE} \
+#    "執行反安裝程式失敗。"
 
-LangString str_msg_abort_install ${LANG_TRADCHINESE} \
-    "安裝程式將退出。"
+#LangString str_msg_abort_install ${LANG_TRADCHINESE} \
+#    "安裝程式將退出。"
 
 LangString str_msg_install_fail  ${LANG_TRADCHINESE} \
     "安裝失敗。預祝下次好運。"
@@ -214,9 +214,9 @@ LangString str_msg_rm_exe_fail   ${LANG_TRADCHINESE} \
     "資料夾「$0」下有部分檔案未能移除！$\r$\n\
      閣下只能手工移除該資料夾。"
 
-LangString str_msg_rm_root_fail  ${LANG_TRADCHINESE} \
-    "警告：無法刪除 Vim 安裝資料夾「$vim_install_root」，\
-     該資料夾下仍有其他檔案。"
+#LangString str_msg_rm_root_fail  ${LANG_TRADCHINESE} \
+#    "警告：無法刪除 Vim 安裝資料夾「$vim_install_root」，\
+#     該資料夾下仍有其他檔案。"
 
 LangString str_msg_wait_uninst   ${LANG_TRADCHINESE} \
     "Waiting the old version to be uninstalled..."
-- 
2.17.0

